libretro/libretro-core.c

```diff
@@ -542,39 +535,39 @@ void retro_set_environment(retro_environment_t cb)
 #if defined(__VIC20__)
       {
          "vice_vic20_external_palette",
-         "External palette; none|mike-ntsc|mike-pal|colodore_vic|vice",
+         "External palette; None|Mike NTSC|Mike PAL|Colodore VIC|Vice",
       },
 #elif defined(__PLUS4__) 
       {
          "vice_plus4_external_palette",
-         "External palette; none|yape-pal|yape-ntsc|colodore_ted",
+         "External palette; None|Yape PAL|Yape NTSC|Colodore TED",
       },
 #elif defined(__PET__)
       {
          "vice_pet_external_palette",
-         "External palette; none|amber|green|white",
+         "External palette; None|Amber|Green|White",
       },
 #else

```

Comment:
Add CBM2 palette handling here. Same as __PET__ but var name should be `vice_cbm2_external_palette`